examples/chained_wizard.py

```diff
@@ -17,10 +17,11 @@
 # Put the Enthought library on the Python path.
 sys.path.append(os.path.abspath(r"..\..\.."))
 
+from traits.api import HasTraits, Int, Str
```

Comment:
Not strictly related to this PR, but `Int` looks like an unused import here.

Reply:
Same here - thought about it, but separate PR.  I'll add an issue.

Reply:
See #1045